Underanalyzer/Decompiler/GlobalFunctions.cs

```diff
@@ -60,29 +72,29 @@ public GlobalFunctions(IEnumerable<IGMCode> globalScripts, ParallelOptions paral
             List<Fragment> fragments = Fragment.FindFragments(script, blocks);
 
             // Find names of functions after each fragment
-            for (int i = 1; i < fragments.Count; i++)
+            foreach (Fragment fragment in fragments)
             {
-                Fragment fragment = fragments[i];
                 if (fragment.Successors.Count == 0)
                 {
                     // If no successors, assume code is corrupt and don't consider it
+                    // TODO: warn?
```

Comment:
So in terms of warnings here, it's a bit weird to somehow get that data out, so I opted to just ignore any code entries that don't resolve global functions. The consequence of this code failing is that function names end up using fully-qualified `gml_Script_` names, rather than their original names (this isn't a *huge* issue, and is actually how DR chapter 2's code was decompiled in UTMT shortly after it came out).

Still, we could have a warning system, maybe. I don't think it would be useful to have details on the specific entry failing (since we could easily debug it), so maybe a boolean specifying whether any entries/fragments failed would work.